aws/modules/core/subnets.tf

```diff
@@ -1,61 +0,0 @@
-resource "aws_subnet" "public" {
-  vpc_id = "${aws_vpc.registers.id}"
-  count = "${length(var.public_cidr_blocks)}"
-  cidr_block = "${element(var.public_cidr_blocks, count.index)}"
-  availability_zone = "${lookup(var.zones, count.index)}"
-
-  tags = {
-    Name = "${var.vpc_name}-public-${count.index+1}"
-    Environment = "${var.vpc_name}"
-  }
-}
-
-resource "aws_route_table" "public" {
-  vpc_id = "${aws_vpc.registers.id}"
-
-  route = {
-    cidr_block = "0.0.0.0/0"
-    gateway_id = "${aws_internet_gateway.igw.id}"
-  }
-
-  tags = {
-    Name = "${var.vpc_name}-public-gateway"
-    Environment = "${var.vpc_name}"
-  }
-}
-
-resource "aws_route_table_association" "public" {
-  count = "${length(var.public_cidr_blocks)}"
-  subnet_id = "${element(aws_subnet.public.*.id, count.index)}"
-  route_table_id = "${aws_route_table.public.id}"
-}
-
-// Create default route with NAT Gateway
-resource "aws_route_table" "private" {
-  vpc_id = "${aws_vpc.registers.id}"
-
-  route = {
-    cidr_block = "0.0.0.0/0"
-    nat_gateway_id = "${aws_nat_gateway.natgw.id}"
-  }
-
-  tags = {
-    Name = "${var.vpc_name}-private-gateway"
-    Environment = "${var.vpc_name}"
-  }
-}
-
-// Endpoint to access S3
-
-resource "aws_vpc_endpoint" "private-s3" {
```

Comment:
Note, I believe this is just the VPC endpoint to access the buckets not the buckets themselves. See https://aws.amazon.com/blogs/aws/new-vpc-endpoint-for-amazon-s3/